internal/output/output.go

```diff
@@ -0,0 +1,54 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/rodaine/table"
+	"gopkg.in/yaml.v3"
+)
+
+func CLIPrint(w io.Writer, format string, data interface{}, headers []any, rowData [][]any) error {
+	switch format {
+	case "yaml":
+		return printYAML(w, data)
+	case "json":
+		return printJSON(w, data)
```

Comment:
We'll want @scotwells to chime in on the marshaling here, as I believe he had plans to migrate away from the GraphQL endpoints and rely on gRPC once they're available.

This would have us marshaling proto structs and would demand using the proto-provided marshalers to handle some of the specialized marshaling logic that proto has.

Reply:
Ah, I just realized @scotwells was already using the proto-json marshaller. My bad. I will update to use the appropriate marshaller here.

Reply:
@joshlreese just fixed so its using the proper proto marshalling. I went on autopilot when writing the marshal funcs 🤦